src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,125 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_LENGTH_ARRAY = 10;
+    public static final int STARTING_ARRAY_INDEX = 0;
```

Comment:
for current task you don't need to add this constant, because it's make harder to read you solution.